batch_job/src/work_queue_factory.c

```diff
@@ -322,7 +324,7 @@ static void set_worker_resources_options( struct batch_queue *queue )
 static int submit_worker( struct batch_queue *queue )
 {
 	char *cmd;
-	const char *worker = "./work_queue_worker";
+	const char *worker = os ? "work_queue_worker": "./work_queue_worker";
```

Comment:
This assumes the worker is always in the PATH in os. Is this always true?

Reply:
if it's in the case of being inside of singularity (as all cctools are in the path of my custom images), or now as being inside the wrapper of vc3-builder, then yes.